.ci/openshift-ci/README.md

```diff
@@ -0,0 +1,63 @@
+# OpenShift CI files
+
+The OpenShift CI (openshift-ci) allows for components to be built and tested in clusters provided by the OpenShift community. An entitled component should follow the [onboarding process](https://github.com/openshift/ci-tools/blob/master/ONBOARD.md) which involve its configuration in the [ci-operator](https://github.com/openshift/ci-operator/blob/master/README.md). So this directory contains scripts and files used by the OpenShift ci-operator to test kata-containers in that CI environment.
```

Comment:
"ci-operator" isn't a dictionary word to you either need to add it to our spell checker dictionary (see https://github.com/kata-containers/tests/tree/master/cmd/check-spelling), or -- simpler -- just refer to it as "CI operator" instead maybe?

Reply:
Yes, I changed the README to use "CI operator" instead.